RFC/src/RFC-0201_TariScript.md

```diff
@@ -507,8 +506,10 @@ For Bob to claim his commitment, \\( C_b \\) he requires the blinding factor, \\
 
 In this case, the script hash is analogous to an address in Bitcoin or Monero. Bob's wallet can scan the blockchain
 looking for hashes that he would know how to resolve. For all outputs that he discovers this way, Bob would need to know
-who the sender is so that he can derive the shared secret.  This information can be obtained without needing to communicate
-with the sender, if she uses the offset public key, \\( K_{Ob} \\) as the key in deriving the Diffie-Hellman exchange.
+who the sender is so that he can derive the shared secret.  
+
+This information can be obtained without needing to communicate with the sender. If the blinding factor \\( k_b \\) was calculated with diffie-hellman using the offset public keypair, (\\( k_{Ob} \\),\\( K_{Ob} \\)) as sender keypair and
+the keypair, (\\( k_{Sb} \\),\\( K_{Sb} \\)) as the receiver keypair, the blinding factor \\( k_b \\) can be securely calculated without communication between the 2 parties.
```

Comment:
Not super necessary, but might be clearer to be 100% specific. 
i.e.
The sender calculates k_b = k_Ob * K_Sb, and the receiver calculates k_b = K_Ob * k_Sb

Reply:
Agreed

Reply:
How does the receiver know the value though?

Reply:
The value must be sent out of band.

Reply:
if the value is sent out of band, why not send the whole commitment out of band? Isn't the purpose here to not do out of band? 
"This information can be obtained with need to communicate with the sender"